MapboxCoreNavigation/NavigationLocationManager.swift

```diff
@@ -15,6 +15,14 @@ import UIKit
 open class NavigationLocationManager: CLLocationManager {
     
     var lastKnownLocation: CLLocation?
+    override open var delegate: CLLocationManagerDelegate? {
```

Comment:
nit: no need to override the default implementation for delegate with the same implementation as the superclass already provides.